### PR TITLE
Nyomi mackey; WIP

### PR DESCRIPTION
**Pull request by nyomilm**

### CODE REVIEW

File: assignments/prototypes.js [hunk 1]
```diff
@@ -15,6 +15,7 @@
   * dimensions (These represent the character's size in the video game)
   * destroy() // prototype method that returns: `${this.name} was removed from the game.`
 */
+function GameObject()
```
Comment (AsaOfDiamonds) [inline assignments/prototypes.js:18]: make sure you actually create the prototype and object, like this
function GameObject(options) {
  this.createdAt = options.createdAt;
  this.dimensions = options.dimensions;
  this.name = options.name;
}

GameObject.prototype.destroy = function () {
  return `${this.name} was removed from the game.`;
};

/*
=== CharacterStats ===
* healthPoints
* name
* takeDamage() // prototype method -> returns the string '<object name> took damage.'
* should inherit destroy() from GameObject's prototype
*/

function CharacterStats(characterStatsOptions) {
  GameObject.call(this, characterStatsOptions);
  this.healthPoints = characterStatsOptions.healthPoints;
}

// Sets up inheritance with GameObject
CharacterStats.prototype = Object.create(GameObject.prototype);

CharacterStats.prototype.takeDamage = function () {
  return `${this.name} took damage.`;
};

Comment (AsaOfDiamonds) [reply assignments/prototypes.js:18]: function Humanoid(humanoidOptions) {
  CharacterStats.call(this, humanoidOptions);
  this.team = humanoidOptions.team;
  this.weapons = humanoidOptions.weapons;
  this.language = humanoidOptions.language;
}

Humanoid.prototype = Object.create(CharacterStats.prototype);

Humanoid.prototype.greet = function () {
  return `${this.name} offers a greeting in ${this.language}.`;
};

**Review by AsaOfDiamonds — APPROVED** [review]
make sure you refer to the solution code and see if you can complete this so you have something to refer to later

**Comment by nyomilm** [comment]
complete

**Pull request merged**